library/src/main/java/com/getstream/sdk/chat/view/MessageInputView.java

```diff
@@ -475,14 +475,14 @@ private void editMessage(Message message) {
         if (attachment.getType().equals(ModelType.attach_file)) {
             String fileType = attachment.getMime_type();
             if (fileType.equals(ModelType.attach_mime_mov) ||
-                    fileType.equals(ModelType.attach_mime_mp4)) {
+                    fileType.equals(ModelType.attach_mime_mp4))
                 messageInputController.onClickOpenSelectMediaView(null, message.getAttachments());
-            } else {
+            else
                 messageInputController.onClickOpenSelectFileView(null, message.getAttachments());
-            }
         } else {
             messageInputController.onClickOpenSelectMediaView(null, message.getAttachments());
         }
+        message.setAttachments(null);
```

Comment:
weird approach (hack?); looks like you are not cleaning up afterwards. What if you edit the message (attachments is reset to null) and then cancel the edit?